### DISCUSSION TITLE
Any issues with xz zipped traces ?

### DISCUSSION THREAD

**Discussion by rvnonsec** [accel-sim/accel-sim-framework#349] [Q&A]
By just changing the one line on the tracer_tool.cu file for nvbit and enabling compressed traces, 
(+  GET_VAR_INT(xz_compress_trace, "TRACE_FILE_COMPRESS", 1,)

Looks like we can save a bunch of disk space (and likely time) if we can provision 1 extra cpu core for xz. 

Everything seems to be already built to work transparently , from tracing, to post processing, to even accel sim runs. 

Is there a reason that compression isnt enabled by default in the repo and automation?

**Comment by JRPan** [comment]
We added that fair recently. We've fixed several bugs so far and it has been quite stable. We are planning to use that by default in the next release. 

It should be ready. If there is any bug, it will be a big one. It will either run or fail. So if it works, you don't have to worry. It won't affect your stats.
